collect-commit-labels.js

```diff
@@ -19,12 +19,12 @@ function collectCommitLabels (list, callback) {
   }
 
   ghauth(authOptions, (err, authData) => {
-    const cache = {}
-
     if (err) {
       return callback(err)
     }
 
+    const cache = {}
```

Comment:
Not related to this patch, but have you thought about using a Set for cache?

Reply:
nope, I don't think I implemented the cache here so it's mostly been out of mind, you're welcome to try and improve it